--- a/src/quickshell/bar/modules/TimeDateWidget.qml	2026-09-06 12:53:52.161985420 +0200
+++ b/src/quickshell/bar/modules/TimeDateWidget.qml	2026-09-06 12:53:52.196186834 +0200
@@ -135,21 +135,19 @@
 
         Column {
             id: timeCol
-            anchors.left: parent.left
-            anchors.leftMargin: timeDateRoot.horizontalPadding
-            anchors.verticalCenter: parent.verticalCenter
+            anchors.centerIn: parent
             spacing: -2
 
             Text {
-                anchors.left: parent.left
+                anchors.horizontalCenter: parent.horizontalCenter
                 text: timeStr
                 font.family: ThemeBackend.fontFamily
                 font.pixelSize: barWindow ? barWindow.s(timeDateRoot.isCompact ? 14 : 15) : (timeDateRoot.isCompact ? 14 : 15)
                 font.weight: Font.Black
-                color: timeDateRoot.isCompact ? Qt.lighter(ThemeBackend.blue, 1.1) : ThemeBackend.blue
+                color: ThemeBackend.text
             }
             Text {
-                anchors.left: parent.left
+                anchors.horizontalCenter: parent.horizontalCenter
                 text: dateStr
                 font.family: ThemeBackend.fontFamily
                 font.pixelSize: barWindow ? barWindow.s(timeDateRoot.isCompact ? 9 : 10) : (timeDateRoot.isCompact ? 9 : 10)
